project/sites/cosicani/themes/cosicani_theme/templates/block/favicon.html

```diff
@@ -0,0 +1,11 @@
+<html><head><link rel="apple-touch-icon" sizes="180x180" href="/sites/cosicani/themes/cosicani_theme/images/favicons/apple-touch-icon.png">
```

Comment:
You need to remove the `<html><head>` from this. I think this has been fixed in Unity_theme so not sure why it is still happening here?

Reply:
Done